### ISSUE TITLE
🔤 Remove paragraph from accessibility statement

### ISSUE THREAD

**Issue by NienkeBr** [GCTC-NTGC/gc-digital-talent#11795]
## 🔤 Copy

The accessibility statement includes a paragraph about collaborating Fable Tech Labs. As we're currently not working with them anymore, this should be removed.

## 🕵️ Details

https://talent.canada.ca/en/accessibility-statement 

![image](https://github.com/user-attachments/assets/0ce2b278-92e0-45ba-879a-2f3ac295cbca)

## 🌎 Localization

Please also remove the corresponding paragraph on the French side

![image](https://github.com/user-attachments/assets/cc38adbd-ef31-44a8-9fc1-27eb54a57d78)

## ✅ Acceptance Criteria

- [ ] Paragraph in accessibility statement about working with Fable Tech is removed, both on English and French pages.
